backend/src/main/resources/application-azure-test.yaml

```diff
@@ -6,6 +6,7 @@ simple-report:
     upload-url: "https://staging.prime.cdc.gov/api/reports?option=SkipInvalidItems" # same destination as dev
     upload-schedule: "0 0 5-21/2 * * *" # hourly on the :00 to differentiate from dev, which sends on the :30
   patient-link-url: https://test.simplereport.gov/app/pxp?plid=
+  twilio-callback-url: https://test.simplereport.gov/api/pxp/callback
```

Comment:
Does this need to be added to Azure config files for other environments?

Reply:
Great call, yes